### ISSUE TITLE
Follow up questions

### ISSUE THREAD

**Issue by akontovasilakos** [nlevos/cat-lovers#1]
Hello! 

Thanks a lot for taking the time to submit this assignment, we really appreciate that.

While reviewing your code, I came up with a couple of questions and would be great if you could find some time to answer:
- I see that you have used inline styling in your components. What is your exposure/experience with other styling approaches (ex. SASS, CSS-in-JS, etc)?
- Since you haven't included any tests in the assignment, I would like to know if you have any experience with any UI testing frameworks. Do you use any particular library or framework when testing your components?

Again, thank you for your effort and time.

**Comment by nlevos** [comment]
Hi Alexandre!

Thank you for taking the time to respond.

Now, to answer your questions,
1. I usually do not use inline styling. (I was trying to be fast - did not
have much time and I forgot to clean up the code). I usually
use css-modules and css stylesheets... I didn't have the opportunity to
work with SASS, as the projects I was involved in had specific requirements
and used specific frameworks that provide their css. Like in the latest
project (ecomsolo) I had to use Shopify Polaris which provides shopify's
css...
2. I have experience writing tests in Jest, but not too much as on the
projects I was involved in I had to do almost everything alone and I had to
move and deliver fast, so
I didn't have time to follow all the steps in the best possible way. I am
comfortable in writing tests with jest, axios-mock-adapter,... I have more
experience in writing tests in other frameworks like in .Net with Nunit and
Xunit...

I would also like to add, as I understand that this is a React role, that I
am very adaptable and fast on getting into things. So if the project
requires SASS, testing or any other
specific architecture, I believe that I will not have any problem getting
up to speed fast, providing reliable results at the same time.

Thank you again for taking the time to respond on my task,
Talk soon,
Nikos
